ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.15.yaml

```diff
@@ -96,6 +96,28 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace-generation
+- as: azure-ipi-private-network-type-basic-f14
+  cron: 5 0 9,25 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_DS4
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_DS4
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-ipi-private
+- as: azure-ipi-private-network-type-m-accelarated-w-basic-f14
```

Comment:
This job is for covering OCP-68548, right? If yes, shall we explicitly instance type?

Reply:
Normally default instance type setting by installer should support capability `AcceleratedNetworkingEnabled`, so I didn't set instance type here.

Reply:
@jianlinliu thanks for the checking, I think it's better to set instance type explicitly for this job to keep consistent with case step and to avoid using default setting in `ipi-conf-azure` . Updated, please help to review again.

Reply:
lgtm